apiserver/plane/app/views/analytic/advance.py

```
@@ -0,0 +1,399 @@
+            "un_started_work_items",
+            "started_work_items",
+        ]
+
+        email = request.user.email
+
+        # Send serialized data to background task
+        export_analytics_to_csv_email.delay(serialized_data, headers, keys, email, slug)
+
+        return Response(
+            {
+                "message": f"Once the export is ready it will be emailed to you at {str(email)}"
```

Ruff (0.8.2)

Line too long (96 > 88)

(E501)